components/docs-chef-io/content/automate/chef_infra_in_chef_automate.md

```diff
@@ -25,6 +24,13 @@ The list of parameters are as follows:
 | nginx['ssl_protocols']                          | TLSv1.2                                   | TLSv1.2                             | ```[cs_nginx.v1.sys.ngx.http]```<br>```ssl_protocols="TLSv1.2"```      |
 | nginx['worker_connections']                     | 10240                                     | 10240                               | ```[cs_nginx.v1.sys.ngx.events]```<br>```worker_connections=10240``` |
 | nginx['worker_processes']                       | 4                                         | 2                                   | ```[cs_nginx.v1.sys.ngx.main]```<br>```worker_processes=4```     |
+| nginx['gzip']                                   | ON                                        |ON                                   |```[cs_nginz.v1.sys.ngx.http]```<br>```gzip= "on"```              |
+| nginx['gzip_comp_level']                         |2                                          |2                                    |```[cs_nginz.v1.sys.ngx.http]```<br>```gzip_comp_level=2```       |
+| nginx['gzip_http_version']                       |1                                          |1                                    |```[cs_nginz.v1.sys.ngx.http]```<br>```gzip_http_version=1```     |
+| nginx['gzip_types']                              |[ 'text/plain','text/css','application/x-javascript','text/xml', 'application/xml','application/xml+rss','text/javascript','application/json']|[ 'text/plain','text/css','application/x-javascript','text/xml', 'application/xml','application/xml+rss','text/javascript','application/json']|```[cs_nginz.v1.sys.ngx.http]```<br>```gzip_types ="[ 'text/plain','text/css','application/x-javascript','text/xml', 'application/xml','application/xml+rss','text/javascript','application/json']"```            |
+| nginx['keepalive_timeout']                      |65                                         |65                                   |```[cs_nginz.v1.sys.ngx.http]```<br>```keepalive_timeout=65```      |
+| nginx['sendfile']                               |ON                                         |ON                                   |```[cs_nginz.v1.sys.ngx.http]```<br>```sendfile ="on"```            |
+| nginx['ssl_ciphers']                            |                                           |ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:AES256-GCM-SHA384:AES:!aNULL:!eNULL:!EXPORT                                     |```[cs_nginz.v1.sys.ngx.http]```<br>```ssl_ciphers=""```            |
```

Comment:
do we need to mention this here. @kalroy , do we have any other option.

Reply:
We should shorten it

```suggestion
| nginx['ssl_ciphers']                            | Refer [nginx['ssl_cipgers'](https://docs.chef.io/server/config_rb_server/#nginx-ssl-protocols-14)                                          |  Refer [nginx['ssl_cipgers'](https://docs.chef.io/server/config_rb_server/#nginx-ssl-protocols-14)                              |```[cs_nginz.v1.sys.ngx.http]```<br>```ssl_ciphers=""```            |
```